practise-mkdir/solution.sh

```diff
@@ -1,2 +1,3 @@
 #!/usr/bin/env sh
 # 请在这里编写一条命令，在当前目录下创建src/main/java/com/github/hcsp目录
+mkdir src/main/java/com/github/hcsp
```

Comment:
```suggestion
mkdir -p src/main/java/com/github/hcsp
```

`-p`参数自动新建所有不存在的祖先目录